docs/spec_v5.md

```diff
@@ -151,30 +148,20 @@ Content-Length: 27
 {"message":"Hello, World!"}
 ```
 
-## Errors
+## Twirp Errors
 
-Twirp error responses are always JSON-encoded, regardless of
-the request's Content-Type, with a corresponding
-`Content-Type: application/json` header. This ensures that
-the errors are human-readable in any setting.
-
-Twirp errors are a JSON object with three keys:
-
-* **code**: One of the Twirp error codes as a string.
-* **msg**: A human-readable message describing the error
-  as a string.
-* **meta**: An object with string keys and values holding
-  arbitrary additional metadata describing the error.
+Twirp responses may also be non-200 errors. Twirp errors are
+JSON responses with the keys `code`, `msg` and (optional) `meta`.
 
 Example:
-```
+```json
 {
-    "code": "permission_denied",
-    "msg": "thou shall not pass",
-    "meta": {
-        "target": "Balrog"
-    }
+  "code": "permission_denied",
+  "msg": "thou shall not pass",
+  "meta": {
+    "target": "Balrog"
+  }
 }
 ```
 
-For more information, see https://github.com/twitchtv/twirp/wiki/Errors.
+For more information see the [Errors Spec](spec_v5_errors.md).
```

Comment:
I think I'd like to have the whole spec live in one document. Could you move the list of error codes and the status code mapping into this file, and delete `doc/spec_v5_errors.md`?

Reply:
Can we do that as a next step? (it will be an easy followup PR). This way we keep this PR clean with the comments in place.

Reply:
No, I'd like to do this in one go. It's better if future readers can see the changes to the spec (and discussions about it) by looking at the git history of just the one file.

I don't think it should be too noisy - it will mostly be adding text, not editing any, right?

Reply:
Yes that's right. Will merge in next commit 👍